polaris-pipeline/Common/Clients/PdfGeneratorClient.cs

```diff
@@ -45,6 +41,12 @@ public async Task<Stream> ConvertToPdfAsync(Guid correlationId, string cmsAuthVa
             if (fileType == FileType.PDF)
                 return documentStream;
 
+            // For integration testing allow, PDF generation to be disabled
+            // By definition this only applies to non-PDFs
+            // In integration this will only be the HTML files generated by e.g. the PCD HTML files generated from case data
+            if (_configuration.IsConfigSettingEnabled(FeatureFlags.DisableTextExtractorFeatureFlag))
```

Comment:
This needs talking about.  The discussion we had involved the new mock implementing the endpoints so that these clients were still doing realistic work and making http calls out to something that is stubbed.  If nothing else, when in disabled mode these activities should take a realistic amount of time to complete, otherwise we will not experience a true behaviour of how activity functions are used up vs the maxConcurrent... setting.